rt/bat/RUNRTBSAM.BAT

```diff
@@ -0,0 +1,31 @@
+@echo off
```

Comment:
New script - needs bash equivalent. Add to BUILD.BAT so included in automated build

Reply:
Fixed 
* 40d533 adds rt/bat/runrtbsam.bat to BUILD.BAT
* 681129 adds bash/runrtbsam script